mlir/lib/Dialect/MIOpen/Transforms/ThreadwiseToStdlib.cpp

```diff
@@ -246,18 +254,34 @@ struct BufferLoadRewritePattern : public OpRewritePattern<BufferLoadOp> {
         oobDims.push_back(pair.index());
     bool toEmitOobChecks = !oobDims.empty();
 
-    auto emitLoadInstruction = [&b, loc, loadedType,
+    auto emitLoadInstruction = [&b, loc, loadedType, sourceType,
```

Comment:
We can get rid of `sourceType` now?

Reply:
Oops you are right, thanks for the reminder